e2e/cypress/plugins/smtpTester.js

```diff
@@ -0,0 +1,34 @@
+const ms = require('smtp-tester');
+
+const initialize = (on, config) => {
+  // starts the SMTP server at localhost:7777
+  const port = 7777;
+  const mailServer = ms.init(port);
+  console.log('mail server at port %d', port);
+
+  let lastEmail = {};
```

Comment:
```suggestion
  const lastEmail = {};
```

Reply:
so, this is an object with the last email per recipient?

Reply:
yes